eos-update-flatpak-repos

```diff
@@ -153,6 +154,55 @@ def _remove_runtimes():
                                '--runtime', refspec])
 
 
+# From 3.7.6 through 3.7.8, images were shipped with remote URLs using
+# our internal ostree.endlessm-sf.com domain instead of our
+# ostree.endlessm.com CDN domain.
+#
+# https://phabricator.endlessm.com/T29535
+def _fix_url(url):
+    parts = urlparse(url)
+    if parts.netloc == 'ostree.endlessm-sf.com':
+        parts = parts._replace(netloc='ostree.endlessm.com')
+        url = urlunparse(parts)
+    return url
+
+
+def _fix_remote_urls():
+    _, repo = OSTree.Sysroot().get_repo()
+    repo.open()
+    for remote in repo.remote_list():
+        # Only adjust non-flatpak remotes (xa.disable = true)
+        _, is_os_repo = repo.get_remote_boolean_option(remote, 'xa.disable',
+                                                       False)
+        if not is_os_repo:
```

Comment:
The OS repo is always called `eos`, right, so perhaps we could simplify this?

Reply:
What we have here is probably fine though.

Reply:
True. I was thinking it could potentially be renamed like a git repo, but ostree doesn't support that. This is now generic, though. I can go either way.